GOVERNANCE.md

```diff
@@ -1,45 +1,117 @@
+### Users
 
-## Community Membership
+Users are community members who have a need for the project. They are the most important members of the community and without them the project would have no purpose. Anyone can be a user; there are no special requirements.
 
-Refer to [Community Membership](community-membership.md).
+The project asks its users to participate in the project and community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user contributions include (but are not limited to):
 
-## Community Structure
+- evangelising about the project (e.g. a link on a website and word-of-mouth awareness raising)
+- informing developers of strengths and weaknesses from a new user perspective
+- providing moral support
+- providing financial support
 
-![community organization](/media/governance/community_organization.png)
+Users who continue to engage with the project and its community will often become more and more involved. Such users may find themselves becoming contributors, as described in the above section.
 
-The community structure comprises two parts. The first part is the community organization, and the second part is the role division within the community group and the promotion path of each community role.
+## Approving PRs
 
-PMC: The Project Management Committee; functions as the core management team that oversees the TiDB community. The PMC decides the roadmap of the project and makes major decisions related to the community.
+Unless otherwise requested by Maintainers, PRs may be merged after receiving at least two approvals from Reviewers or higer roles.
 
-Developer Group:
+## Decision making and voting
 
-- Maintainer: Mainatiners are the planners and designers of the project, with the authority to merge branches into the master. 
-- Committer:  Recommended by Maintainers or the PMC as Contributors with outstanding contributions to TiKV. A Committer must have completed at least one feature or fix a major bug independently.
-- Reviewer: Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Contributors. Reviewer has the rights to approve code reviews with LGTM (Looks Good To Me).
-- Active contributor: Contributed 8 PRs and above to TiKV within one year and received recognition from 2 incumbent Reviewers
-- Contributor: Merged at least 1 PR in any Repo under [TiKV](https://github.com/tikv) 
+Ideally, all project decisions are resolved by consensus via a PR or GitHub issue. If this is not possible, Maintainers may call a vote. Many of the day-to-day project maintenance can be done by a [lazy consensus](http://communitymgt.wikia.com/wiki/Lazy_consensus) model. But the following items must be called to vote:
```

Comment:
I think most decisions can be discussed on GH, but some cannot (e.g., whether to ban a member of the community). I think the discussion venue (GH, private meeting, etc) is orthogonal to the decision making mechanism (voting, consensus, etc).

Reply:
So you are suggesting we add something like a private maintainer meeting for

Reply:
I don't think we need to put it here. If the maintainers want an ad hoc or a regular meeting, then they can schedule one.